docs/developer.md

```diff
@@ -17,4 +17,4 @@ After you have done installing the package, follow these steps:
 
 ## Develop this package for VS Code
 
-Clone [vscode-markdown-preview-enhanced](https://github.com/shd101wyy/vscode-markdown-preview-enhanced), then `npm install`, open in **vscode** and start debugging.
+Clone [vscode-markdown-preview-enhanced](https://github.com/shd101wyy/vscode-markdown-preview-enhanced), then `yarn install`, open in **vscode** and start debugging.
```

Comment:
@kachkaev seems like all `npm install` are replaced with `yarn install`, but they should still be `npm install` or `yarn` instead?

Reply:
The commands in the instructions should be `yarn install` (or just `yarn`). If we tell people to use `npm install`, `yarn.lock` file will be ignored and `package-lock.json` will be created. There is a chance that people will commit that new file by a mistake and this may cause problems. Since all three repos of yours are using `yarn` now, I think it's correct to remove all `npm` commands 🙂

Reply:
Replied in another thread

Reply:
If you prefer, I can replace `yarn install` with `yarn` (which is the same as `yarn install`).

Reply:
`yarn install` is absolutely fine :+1: